### ISSUE TITLE
Can’t turn on Variable Font Preview

### ISSUE THREAD

**Issue by harbortype** [Mark2Mark/variable-font-preview#19]
**Edit by Mark:**
See below for the solution.
Tldr: remove the "paddle-framework" from the repositories folder manually restart Glyphs and install the plugin again.

---

**Describe the bug**
Whenever I try to activate/turn on the Variable Font Preview from the View menu, it just flashes on the screen and immediately deactivates itself.

**To Reproduce**
1. Click on the menu item View > Show Variable Font Preview 3

**Expected behavior**
The plugin remains active.

**Screenshots**
https://user-images.githubusercontent.com/10968309/189368113-2279e9b4-6d3b-428d-89c3-65ab60967068.mov

**Desktop (please complete the following information):**
 - macOS: Big Sur 11.6.8
 - GlyphsApp Version build number 3135
 - Version (not sure)

**Additional context**
I see this happened before due to an incompatibility with Red Arrows (#8), but in this case no other plugin is active. Also, this behavior occurs only on my 2014 MacBook Pro. My main computer is a 2018 Mac mini, in which it runs fine.
Nothing is shown on the Macro Window.
Console.app shows this the moment I activate the plugin:
```
default	11:08:57.210038-0300	mDNSResponder	[R2751] DNSServiceQueryRecord(1D000, 0, <mask.hash: 'Vhn6SOOmFqxKow5N1N4zLw=='>, Addr) START PID[20589](Glyphs 3)
default	11:08:57.212887-0300	mDNSResponder	[R2752] DNSServiceQueryRecord(1D000, 0, <mask.hash: 'Vhn6SOOmFqxKow5N1N4zLw=='>, AAAA) START PID[20589](Glyphs 3)
default	11:08:57.216872-0300	mDNSResponder	[R2751] DNSServiceQueryRecord(1D000, 0, <mask.hash: 'UuQ/n5lmoaUfPEY4oJEWwg=='>, Addr) STOP PID[20589](Glyphs 3)
default	11:08:57.219089-0300	mDNSResponder	[R2752] DNSServiceQueryRecord(1D000, 0, <mask.hash: 'UuQ/n5lmoaUfPEY4oJEWwg=='>, AAAA) STOP PID[20589](Glyphs 3)
```
I believe it might have something to do with the license activation, but I don’t know what to do about it because I can’t get to the license window. I tried removing the plugin and reinstalling it, but nothing changed.

**Comment by Mark2Mark** [comment]
Hi and thanks for the detailed report.

So it is happening only on the very old mac, which is also not your main machine?
May I ask how important it is for you to run on that old mac? It seems like a very hard thing to debug, if it is indeed related to the old (9 years) hardware.

**Comment by FriedrichAlthausen** [comment]
Hi @Mark2Mark, 
I am having the exact same effect that prevents me from entering my freshly purchased license key. 
The plugin deactivates itself after only flashing once. No chance to access or even see any popup window. 
Glyphs 3140 on MacOS 12.2.1
Thanks in advance!

**Comment by Mark2Mark** [comment]
Hi @FriedrichAlthausen,

Thanks! I'll have a look in a bit. Sorry for the inconvenience to both of you!

**Comment by harbortype** [comment]
No problem! It’s been a while since I last used my MBP. I just learned a couple of things:

1. Other users on the same machine are unaffected.
2. Console.app displays a couple of errors when Glyphs is starting up which might be relevant:

<img width="1134" alt="Screen Shot 2022-10-22 at 16 38 30" src="https://user-images.githubusercontent.com/10968309/197359478-7dc32a91-e148-48de-ab2d-076a97bd04ef.png">

Let me know if you need anything else.

**Comment by Mark2Mark** [comment]
This error message looks like the plugin manager is not correctly downloading the Paddle Module (which it should).

I need to investigate. Thanks, @harbortype

**Comment by Mark2Mark** [comment]
> 1. Other users on the same machine are unaffected.

@schriftgestalt, do you know why the Paddle module seems to not get loaded for only a certain user of the machine?
(Note, these are the only 2 users having an issue)

**Comment by Mark2Mark** [comment]
@FriedrichAlthausen can you see the same thing in the Mac System Console as in the screenshot by harbourtype?

**Comment by Mark2Mark** [comment]
> 1. Other users on the same machine are unaffected.

@harbortype so one user can run the plugin as expected, but when you switch to another mac user profile, this happens? which user works and wich fails? Is it an admin and a guest kind of situation?

**Comment by FriedrichAlthausen** [comment]
> @FriedrichAlthausen can you see the same thing in the Mac System Console as in the screenshot by harbourtype?

Hi @Mark2Mark , I cannot find a similar message in Console.app. 
(Maybe I am just not too familiar with reading these errors …?)

**Comment by Mark2Mark** [comment]
Thanks!

Can both of you please have a look into the "Repositories" folder (right next to the "Plugins" folder) and check if there is a folder called "paddle-framework"?

It should be somewhere like `/Users/{USERNAME}/Library/Application Support/Glyphs 3/Repositories/paddle-framework`

If it is there, please delete it, restart Glyphs and install the plugin again, restart Glyphs. 

Let me know if this
a) helped
b) did not help
c) that folder was not even there

Thanks so much for your patience and help @harbortype  @FriedrichAlthausen !

**Comment by harbortype** [comment]
> If it is there, please delete this "paddle-framework" folder, restart Glyphs and install the plugin again, restart Glyphs.

It worked! There was a `paddle-framework` in `Repositories` which was empty except for a `.git` subfolder. I followed your steps and everything is working now. Thank you so much!

> @harbortype so one user can run the plugin as expected, but when you switch to another mac user profile, this happens? which user works and wich fails? Is it an admin and a guest kind of situation?

Just to let you know, it didn’t work on my user and it did on my wife’s. Both users are administrators.

**Comment by FriedrichAlthausen** [comment]
Thanks @Mark2Mark ,
I can report a). 
That solved it for me!
Thank you a lot for this plugin!

**Comment by Mark2Mark** [comment]
Hi @harbortype @FriedrichAlthausen 

Did anything of that ever occur again? Otherwise I want to close this issue.

Many thanks!

**Comment by harbortype** [comment]
It’s all good for me. Thanks!
